src/main/asciidoc/en/Ch04_Listener.adoc

```diff
@@ -524,3 +524,90 @@ Listener interface::
 Annotation::
   It is used when business logic specific process is to be performed. +
   As a rule, it is implemented only for ItemProcessor.
+  
+[[Ch04_Listener_HowToUse_JobAbort]]
+=== Job abort in pre-processing(StepExecutionListener#beforeStep())
+When the conditions for executing the job are not satisfied, you may want to abort the process before executing the job.
+
+In such a case, the process can be aborted before job execution by throwing an exception in pre-processing (``beforeStep`` method). +
+The case of where the following requirements are implemented is explained with the help of an example here.
+
+. Verify the validity of start parameters of input and output files by ``beforeStep`` method defined by ``StepExecutionListener``. +
+. Throw an exception if any of the start parameters is not specified.
+
+However, it is recommended to use ``JobParametersValidator`` for verifying the validity of start parameters in {batch5_shortname}.
```

Comment:
@bttendulkara L-538
Please modify as -
However, in {batch5_shortname}, it is recommended to use ``JobParametersValidator`` for verifying the validity of start parameters.